java.exercises/src/test/Ampeg Manual Test Project.txt

```diff
@@ -0,0 +1,117 @@
+--Ampeg Manual Test Project--
+https://ampeg.com/
+
+1. You decide what should be priority (to test) in the webapp.
+2. 6 manual test cases total.
+3. Ensure to include what 'should' be included in a manual test (title, summary, etc).
+4. Ensure that 5 tests consist of navigating to more than one page.
+5. Ensure that 2 tests consist of navigating to 3 or more pages.
+
+Title
+Summary
+Pre-Conditions
+Steps
+Expected Result
+
+---------------------------------------------------------------------------------------------
+
+Title 	: 001_SCR-DI_buy_test
```

Comment:
Use plain English titles - Can buy products using Buy link
The ID will be stored in a separate field.